include/aws/cryptosdk/cipher.h

```diff
@@ -45,6 +45,10 @@ enum aws_cryptosdk_rsa_padding_mode {
     AWS_CRYPTOSDK_RSA_OAEP_SHA256_MGF1,
 };
 
+/* This is large enough to hold an encoded public key for all currently supported curves */
+#define MAX_PUBKEY_SIZE 64
+#define MAX_PUBKEY_SIZE_B64 (((MAX_PUBKEY_SIZE + 2) * 4) / 3)
```

Comment:
Where does this math come from

Reply:
I just moved these defines from the `cipher_openssl.c` file, so they can be used in the proofs.